recipes/gridgen/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib"
+export CPPFLAGS="-I/$PREFIX/include $CPPFLAGS"
+
+pushd gridgen
+
+./configure --prefix=$PREFIX
+
+make
+make lib
+make shlib
```

Comment:
@jakirkham 

Yes (static and shared libs).
 
In the early days of me using these libraries, the incantations make everything link properly involved a little bit of magic, making sure the handles of the pots and pans in your kitchen were pointed north, and a little bit of luck. 

Unfortunately, `autoconf`, configure scripts, etc, are very much a black box to me. What you see above is the first procedure I found that reliable built everything. One it worked, I left it alone.

In short, this was me flailing around until everything worked.